### ISSUE TITLE
remove-completed missions no longer works in build 51

### ISSUE THREAD

**Issue by alexnpark** [OmerBenGera/SuperiorSkyblock2#52]
Hi Omer,

Thanks for fixing that bug with the required-checks- It appears that after updating to build **51**, the option to remove-completed-missions from the GUI no longer works. When a mission is completed, it remains in the GUI despite having the option `remove-completed: true` set to true.

Spigot Version: 1.16.4, 1.16.5 (Tested with Spigot, Paper, and Tuinity)
Plugin Version:. SuperiorSkyblock2-51

**Comment by OmerBenGera** [comment]
Check out build 53., let me know if it fixes the issue

**Comment by alexnpark** [comment]
Hey, it seems that there are now no missions showing in the GUI at all. I loaded up the default missions.yml just to be sure there wasn't anything wrong in my own file, but the GUI Is still completely blank.

**Comment by OmerBenGera** [comment]
Which mission menu are you even trying to check?

**Comment by OmerBenGera** [comment]
I've just tried to replicate the issue and it seems to work fine for me - all the missions are there, besides the ones I already completed

**Comment by alexnpark** [comment]
Hey Omer, after some testing, I've pinned down what's actually happening for me. It appears that the setting is doing the opposite, and when I toggle `remove-completed:` to `true`, it makes the mission _appear_ in the GUI instead of disappear. So before you complete the mission, the entire GUI is empty, then if you complete the mission via command, it appears into the GUI.

I've attached a video for clarity. I'm testing this with player missions, have reset the player-missions menu to default, as well as the actual missions.yml file to default as well. It's quite strange this isn't reproducible since I'm using default everything, so let me know if this helps at all.


[https://www.youtube.com/watch?v=BB4r5Q-_L_w&feature=youtu.be](https://www.youtube.com/watch?v=BB4r5Q-_L_w&feature=youtu.be)
